docs/kyma/03-02-testing.md

```diff
@@ -40,15 +40,19 @@ The following example presents TestDefinition with a container that calls the De
 apiVersion: "testing.kyma-project.io/v1alpha1"
 kind: TestDefinition
 metadata:
-  name: "test-{{ template "fullname" . }}-connection-dex"
+  name: {{ .Chart.Name }}
+  labels:
+    app: {{ .Chart.Name }}-tests
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
```

Comment:
Maybe we can also clean up the labels? Helm [suggestions](https://helm.sh/docs/chart_best_practices/#labels-and-annotations) regarding labels are different.

Reply:
I applied the suggestions. However I still added `app`as functionality might break without.